Common/Data/UniverseSelection/SubscriptionRequest.cs

```diff
@@ -103,7 +103,13 @@ public IEnumerable<DateTime> TradableDays
             Universe = universe;
             Security = security;
             Configuration = configuration;
-            StartTimeUtc = startTimeUtc;
+
+            // open interest data comes in once a day before market open,
+            // make the subscription start from midnight
+            StartTimeUtc = configuration.TickType == TickType.OpenInterest ?
```

Comment:
Suggest we merge this as scaffolding to patch the issue but would be nice to keep brain storming. Perhaps there's a more generic solution? Can we set the start date earlier in the stack?  Could certain subscriptions be full day?